products/extensions/b2b-components/employee-management/concept/README.md

```diff
@@ -0,0 +1,16 @@
+# Concept
+
+## Basic idea
+
+Employee management, as part of the B2B components, is a feature that allows you to manage employees and their permissions as an extension to Shopware's account and customer management, but gathered in a company context. This means that employees are associated with a **company customer** and will act on behalf of that company, when talking about e.g. placing orders in the shop using addresses and payment methods defined by administrators of such company.
+For that matter, we needed a way to provide order relevant data without building the checkout process anew and decided on using a **company customer**
```

Comment:
```suggestion
For that matter, we needed a way to inject company managed data into core processes, without building them from scratch and without maintaining multiple versions of the same processes. This is why we decided on using a **company customer**.
```

Reply:
Meyer and me created a new approach there. I guess that fits. Please check again.